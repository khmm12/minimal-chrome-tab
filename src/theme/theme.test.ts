import { colors, fonts, media } from './index.js'

test('colors', () => {
  expect(colors).toMatchSnapshot()
})
test('fonts', () => {
  expect(fonts).toMatchSnapshot()
})
test('media', () => {
  expect(media).toMatchSnapshot()
})
