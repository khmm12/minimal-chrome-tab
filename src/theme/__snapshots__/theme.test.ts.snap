// Vitest Snapshot v1

exports[`colors 1`] = `
{
  "black": "#000000",
  "dark1": "#202124",
  "lightGrey": "#eeeeee",
  "white": "#ffffff",
  Symbol(Symbol.toStringTag): "Module",
}
`;

exports[`fonts 1`] = `
{
  "Digital7Mono": "Digital-7Mono",
  "HelveticaNeue": "\\"Helvetica Neue\\", Helvetica",
  Symbol(Symbol.toStringTag): "Module",
}
`;

exports[`media 1`] = `
{
  "beforeSmall": "(max-width: 599px)",
  "darkScheme": "(prefers-color-scheme: dark)",
  Symbol(Symbol.toStringTag): "Module",
}
`;
